.github/workflows/backend-snyk-scan.yml

```diff
@@ -0,0 +1,24 @@
+name: snyk scanning
+
+on:
+  push:
+    branches: ["master", "release-*"]
+
+  workflow_dispatch:
```

Comment:
What is this for?

Reply:
Since the CLI scanning won't just be pulling your whole repo, we have to define which branches we want it to scan. In this case we're doing master and release branches but if you don't think that's right we can change it. `workflow_dispatch` just lets us run it manually from the UI if there is a need to, for example if a scan fails because of an API error or something.

Reply:
@wreet Sorry I meant to only highlight the workflow dispatch. Can you explain how that works? Because there's no configuration attached to the `workflow_dispatch` property

Reply:
Ah I see what you mean. That property does not require any additional configuration since we are not passing any inputs to the action.